context/.eslintrc.yml

```diff
@@ -20,3 +20,7 @@ rules:
   react/jsx-one-expression-per-line: [0]  # Makes punctuation after tab awkward.
   react/prop-types: [0]
   import/prefer-default-export: [0]  # Hit eslint error: SyntaxError: Unexpected token, expected {
+  jsx-a11y/label-has-associated-control:
```

Comment:
I think they are already being used, I just had to add this to tell eslint that the `Switch` should count as a control component, in addition to the regular `<input>` elements. Otherwise it complains because it does not know which custom elements wrap control components.